newsletters/20190918-trial-users.md

```diff
@@ -0,0 +1,45 @@
+Hi there!
+
+You're receiving this email because you recently tried the Learnk8s Academy.
+
+_There's some news for you._
+
+**1️⃣ Download your certificate of completion!**
+
+When you complete the Zero to Kubernetes course, you can also apply for a certificate of completion.
+
+There's a catch, though._
+
+The certificate is hidden behind a challenge, and you can reveal it only by solving it.
+
+[Download your certificate →](https://academy.learnk8s.io/dashboard "Download your certificate")
+
+**2️⃣ More hands-on challenges**
+
+If you like getting your hands dirty, you will love the new challenges for the Kubernetes Fundamentals course.
+
+The challenges are designed to help you practice deployments in Kubernetes.
+
+**3️⃣ A better learning experience**
+
+In the past six months you asked for:
+
+- offline access to the Academy
+- tracking progress online
+- no more subscription — a single, one-time fee
+
+Your voice was heard loud and clear.
+
+The Learnk8s Academy is **available as ebooks** so that you can download and study when you're commuting.
+
+You can **track progress as you read** through the content.
+
+And best of all, **no more subscriptions**.
+
+You can buy access to the content and use it how many times you want for as long as you want.
+
+_Interested in getting access to the rest of the material?_
```

Comment:
What does "the rest of the material" refer to?

Maybe:

- _Interested in getting access to the new Academy?_